rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/start_scan/action.py

```diff
@@ -33,8 +33,9 @@ def run(self, params={}):
         body = {"asset_ids": asset_ids, "name": name}
 
         response = self.connection.ivm_cloud_api.call_api("scan", "POST", None, body)
+        get_id = response[1].get("scans")
 
-        return response[1]
+        return {Output.DATA: response[1], Output.ID: get_id[0].get("id")}
```

Comment:
These two lines should be try/excepted for IndexError in the case that they do not have any elements